lib/response_handler.rb

```diff
@@ -1,4 +1,5 @@
 require 'response'
+require 'game'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.